Source/WebCore/platform/graphics/filters/FEComponentTransfer.h

```diff
@@ -55,10 +56,20 @@ class FEComponentTransfer : public FilterEffect {
 public:
     WEBCORE_EXPORT static Ref<FEComponentTransfer> create(const ComponentTransferFunction& redFunc, const ComponentTransferFunction& greenFunc, const ComponentTransferFunction& blueFunc, const ComponentTransferFunction& alphaFunc);
 
-    ComponentTransferFunction redFunction() const { return m_redFunction; }
-    ComponentTransferFunction greenFunction() const { return m_greenFunction; }
-    ComponentTransferFunction blueFunction() const { return m_blueFunction; }
-    ComponentTransferFunction alphaFunction() const { return m_alphaFunction; }
+    enum class Channel : uint8_t { Red, Green, Blue, Alpha };
```

Comment:
Maybe this can be a global enum similar to ComponentTransferType since it difficult to decide whether it belongs to FEComponentTransfer, SVGComponentTransferFunctionElement or SVGFEComponentTransferElement